### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by bhudlemeyer**
When walking through the latest 1.0 beta install hit several inconsistencies in the documentation. Specifically around the quickstart deployment and the continued direction to use IngressRoute over HTTPProxy in the examples. 

I do not have access to update the following location:

https://projectcontour.io/examples/kuard-httpproxy.yaml

Would need assistance getting the attached file uploaded to the correct location to ensure the HTTPProxy documentation functions as expected. 

Git wont allow uploading a yaml so whoever is able to perform upload would need to rename file to .yaml if this pull request is approved. 

[kuard-httpproxy.txt](https://github.com/projectcontour/contour/files/3693555/kuard-httpproxy.txt)

### CODE REVIEW

File: examples/contour/README.md [hunk 1]
```diff
@@ -20,10 +20,9 @@ This configuration has several advantages:
 For detailed instructions on how to configure the required certs manually, see the [step-by-step TLS HOWTO](../../docs/grpc-tls-howto.md).
 
 ## Deploy Contour
-
 Either:
 
-1. Run `kubectl apply -f https://projectcontour.io/quickstart/contour.yaml`
+1. Run `kubectl apply -f https://raw.githubusercontent.com/projectcontour/contour/master/examples/render/contour.yaml
```
Comment (stevesloka) [inline examples/contour/README.md]: The quickstart points to the latest stable release, today that is v0.15. There are some inconsistencies that come up due to this which we are discussing in #1619. 

TLDR: Let's keep this pointing to the quickstart which gets updated each new release of Contour.

File: docs/deploy-options.md [hunk 2]
```diff
@@ -155,7 +191,10 @@ If you can't or don't want to use a Service of `type: LoadBalancer` there are ot
 
 If your cluster doesn't have the capability to configure a Kubernetes LoadBalancer,
 or if you want to configure the load balancer outside Kubernetes,
-you can change the Envoy Service in the  `02-service-envoy.yaml` file to set `type` to `NodePort`.
+you can change the Envoy Service in the `02-service-envoy.yaml` file and set `type` to `NodePort`.
+  * If you have cloned down the projectcontour git repo this file can be found at:
```
Comment (stevesloka) [inline docs/deploy-options.md]: Could we link back to the Envoy service file in the location you specified and remove the phrase about "if you have cloned down the..."?

File: examples/contour/README.md (hunk 1 above)
Comment (bhudlemeyer) [reply examples/contour/README.md]: Should we put a note or something specifying the updated contour.yml if doing a 1.0.0-1beta install until the work in #1619 is completed. As currently even following the docs and links directly from projectcontour.io leads to this same page?

example flow we followed:
https://projectcontour.io/ 
  --> Documentation 
     --> v1.0.0-beta.1 (https://github.com/heptio/contour/tree/v1.0.0-beta.1/docs) 
         --> README.md Deployment Options (https://github.com/projectcontour/contour/blob/v1.0.0-beta.1/docs/deploy-options.md) 
    --> Recommended Installation details Link in paragraph contour example 
          (https://github.com/projectcontour/contour/blob/v1.0.0-beta.1/examples/contour/README.md) 
        --> leads to quickstart link still showing old deployment but all wording on page is updated to call out new deployment method (IE daemon sets for envoy containers).

File: docs/deploy-options.md (hunk 2 above)
Comment (bhudlemeyer) [reply docs/deploy-options.md]: ya no problem will update

File: examples/contour/README.md (hunk 1 above)
Comment (stevesloka) [reply examples/contour/README.md]: I agree this is broken. The issue is there are multiple links that all use this quickstart. We'll look to fix that in #1619 which should happen very soon so that this is a much smoother experience.

Comment (bhudlemeyer) [reply examples/contour/README.md]: doc changed as requested to point back to quickstart

**Comment by stevesloka** [comment]
Thanks for the work @bhudlemeyer! Would you mind now rebasing everything into a single commit then we can merge this!

**Comment by bhudlemeyer** [comment]
Yeap when I get back to personal computer later tonight will rebase.

**Comment by stevesloka** [comment]
Hey, @bhudlemeyer we're sooo close! Could you fix the merge conflict, rebase, then repush? Hopefully, that will also rekick Travis to let the CI build pass. Thanks!

**Comment by bhudlemeyer** [comment]
It says I can't due to not having write access to the repository?

**Comment by bhudlemeyer** [comment]
Actually looks like master removed a bunch of the docs out of the repo for some reason? So me adding docs/deploy-options.md back is causing the breakage. Is this correct that the entire file should be removed? If so I probably need to fix the README.md under examples as it links to that file.

**Comment by bhudlemeyer** [comment]
Looking at whats going on now I could change the test link in examples to point to https://projectcontour.io/getting-started/ 

Instead of the git repo doc? Is this what we would want to do now that the other docs repo in git has been wiped it appears.

**Comment by stevesloka** [comment]
Ok so bad timing on this, we've been moving much of the docs to the project site. The deploy-options.md is now here: https://github.com/projectcontour/contour/blob/master/site/_guides/deploy-options.md

**Comment by bhudlemeyer** [comment]
So I cant seem to get the last rebase to work right after fixing the merge conflict that always shows up with my file located at site/_guide/.

Any pointers on what I need to do to get the rebase to work correct would be appreciated.

As soon as I get the conflict fixed I see around 20+ commits in the rebase and squashing them seems to cause other files to go a bit crazy in my repo on me.

**Comment by davecheney** [comment]
@bhudlemeyer I'll merge this for you. Thank you.
